### DISCUSSION TITLE
Exclude specific collections from movies section

### DISCUSSION THREAD

**Discussion by Kastrel7** [jellyfin/jellyfin#12282] [General]
So, I have a lot of collections on my server, but some of them are collections from studios such as Dreamworks or BlueSky. These collections have more collections inside as well as stand alone movies. 

A great option would be to have the ability to exlude these collections from the movies list so the stand alone movies in the collection get displayed on their own in the movies list, while keeping the collection in the collections list. 

An extra option could also be to keep the collection in the movies list, but also have the movies within the collection in the movies list.

**Comment by TheMelmacian** [comment]
What exactly do you mean? I have many collections with mixed content and each content is sorted in its own category inside the collection:
- Videos
- Shows
- Albums
- Books
- Other content (includes other Collections)

I've never seen that movies and collections are mixed together inside the same list.

**Reply by Kastrel7** [reply]
If you go to Dashboard > Libraries > Display, you have the option to group movies in collections, which adds all your collections into the Movies sections on Jellyfin. If a movie is in a collection, it isn't displayed in the Movies sections anymore and is only available in the collection it is a part of within the Movies section. 

For example, I have a collection of all the DreamWorks films, which includes The Bee Movie, when you go into your Movies section, the only way to find The Bee Movie is by going into the DreamWorks collection and finding it from there.

My problem is that some of the users I give access to my server might not know that The Bee Movie is created by DreamWorks, and won't think to look in the DreamWorks collection, and instead will go to 'B' (if they're sorting in alphabetical order) and look for the movie there. 

The option to exclude a specific collection from the Movies section would solve this problem I'm having and maybe others.

**Reply by TheMelmacian** [reply]
Ah, OK. Never used this option.

**Reply by ChrisBOfficial** [reply]
Seconding this request. Not all of my movies are in 4K, but this would allow me to group the ones that are 4K into their own collection while also keeping the movies visible in the main movie library. Mostly a QoL feature.

**Reply by zlucas96** [reply]
I also want to see this feature. Plex allows you to change this per collection, not just globally.

**Comment by josephrkuhn** [comment]
I'd love to see this feature as well. Similarly I'm setting up collections for Disney films, DreamWorks films, etc. but I'd like certain collections to show movies outside of them as well. I want to set up seasonal collections too but I don't want movies to only appear in those specific collections as it will be harder to see what movies are in the overall library. I don't want my 8 Harry Potter films shown separately, but for certain collections I would really like this option.
